src/views/Users.vue

```diff
@@ -0,0 +1,40 @@
+<template>
+  <div>
+    <h2>
+      Users
+    </h2>
+    <div v-if="loading" class="text-xs-center mt-5">
+      <v-progress-circular
+        :width="1"
+        color="red"
+        :size="70"
+        indeterminate
+      />
+    </div>
+
+    <div v-for="user in userList" v-else :key="user.id" style="display: inline-block">
+      <Preview :user="user" />
```

Comment:
good that you are using props, but dont send all object, send just id, and take rest of information from store